common/src/hand_off.rs

```diff
@@ -261,6 +261,18 @@ pub struct FirmwareHandoffTable {
     /// Index of RT SVN value in the Data Vault
     pub rt_svn_dv_hdl: HandOffDataHandle,
 
+    /// LdevId TBS Size.
```

Comment:
@jhand2 had a suggestion to also propagate addresses, rather than rely on the linker script. Thoughts?

Reply:
Sounds good; I'll add (TBS and LOG) addresses to the FHT and will remove them from the linker script.